docs/source/getting_started.rst

```diff
@@ -22,19 +57,93 @@ You can create and activate the environment by:
 
     $ pip install -e .
 
+++++++++++++++++++++++++++++++++
+Install CORSIKA and sim_telarray
+++++++++++++++++++++++++++++++++
 
-Each time you want to use the package, just activate the conda environment:
+CORSIKA and sim_telarray are external tools to gammasim-tools and are used by several gammasim-tools application.
+Follow the instruction provided by the CORSIKA/sim_telarray authors for installation.
+CTA users can download both packages from `here <https://www.mpi-hd.mpg.de/hfm/CTA/MC/Software/Testing/>`_ and install the package with e.g.:
 
 .. code-block:: console
 
-    $ conda activate gammasim-tools-dev
+    tar -czf corsika7.7_simtelarray.tar.gz
+    ./build_all prod5 qgs2 gsl
+
+
++++++++++++++++++++++++++++
+Set environmental variables
++++++++++++++++++++++++++++
+
+Source the ``set_DB_environ.sh`` script (see `Model Database Access`_) to activate set the environmental variables for the DB access:
+
+.. code-block:: console
+
+    $ source set_DB_environ.sh
+
+The environmental variable ``$SIM_TELPATH`` should point towards the CORSIKA/sim_telarray installation.
+
++++++++++++++++++
+Test installation
++++++++++++++++++
 
-If you need access to the DB, ask one of the developers for the credentials, copy \
-set_env_db_template.sh to a new file named set_env_db.sh, update it with the credentials and \
-source it to activate the environmental variables:
+Test your installation by running the unit tests:
 
 .. code-block:: console
 
-    $ source set_env_db.sh
+    $ pytest tests/unit_tests/
+
```

Comment:
Integration tests take a long time and I am not sure what the user would learn from running them here? The unit tests are a good and complete test of the setup and it is mentioned here to allow the user to be sure that things are fine. So I would prefer not to add it, otherwise the installation procedure will take quite a long time.

Reply:
That is right, but we could indeed test one application that would confirm that the DB connection is working. For example, this test takes 4.7s:
```
pytest tests/integration_tests/test_applications.py -k get_file_from_db::CTA-Simulation-Model
```
We could pack it in a shell script test_DB.sh and run it. But thats only a suggestion